chpl/chpl-resources/src/main/resources-production/user-triggers.xml

```diff
@@ -4,12 +4,7 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://www.quartz-scheduler.org/xml/JobSchedulingData http://www.quartz-scheduler.org/xml/job_scheduling_data_2_0.xsd"
     version="2.0">
-    <pre-processing-commands>
```

Comment:
These pre-processing commands were here for a reason. I don't want to remove them unless we're sure they can be

Reply:
The pre-processing commands are still in `resources/jobs.xml` which is used by all environments.